src/components/organisms/accordion/index.tsx

```diff
@@ -123,12 +48,17 @@ const Item: React.FC<AccordionItemProps> = ({
           <MorphingTrigger />
         </AccordionPrimitive.Trigger>
       </AccordionPrimitive.Header>
-      <AccordionPrimitive.Content forceMount>
-        <div className="text-grey-50 inter-base-regular transition-all duration-300 h-radix-accordion opacity-100 group-radix-state-closed:h-0 overflow-hidden group-radix-state-closed:opacity-0">
-          {description && <p>{description}</p>}
-          <div className="mt-large">{children}</div>
-        </div>
-      </AccordionPrimitive.Content>
+      <div className="group-radix-state-closed:h-0 group-radix-state-closed:opacity-0 group-radix-state-open:h-radix-accordion group-radix-state-open:opacity-100 transition-all duration-300 ease-accordion overflow-hidden">
+        <AccordionPrimitive.Content
+          forceMount
+          className="overflow-hidden radix-state-open:animate-accordion-open radix-state-closed:animate-accordion-close"
+        >
+          <div className="text-grey-50 inter-base-regular">
+            {description && <p>{description}</p>}
+            <div className="mt-large">{children}</div>
+          </div>
+        </AccordionPrimitive.Content>
+      </div>
```

Comment:
I think this can also be solved in the following way, by adding the close animation to the content of the `<AccordionPrimitive.Content>`.

I'm not 100% that this covers all edge cases but i couldn't break it

```suggestion
      <AccordionPrimitive.Content
        forceMount
        className={clsx(
          "overflow-hidden radix-state-open:animate-accordion-open radix-state-closed:animate-accordion-close"
        )}
      >
        <div className="text-grey-50 inter-base-regular group-radix-state-closed:animate-accordion-close">
          {description && <p>{description}</p>}
          <div className="mt-large">{children}</div>
        </div>
      </AccordionPrimitive.Content>
```

Reply:
Works like a charm! Also made forceMount optional as we might not always need it 😄